src/loader/endpoints/get_inloco_cities.py

```diff
@@ -1,51 +1,169 @@
 import pandas as pd
+import sys
+import numpy as np
+
+sys.path.append("..")
+sys.path.append("../src")
 from utils import secrets
 from endpoints.helpers import allow_local
 import endpoints.GoogleDocsLib as gd
-import endpoints.corrections as cr
+import Levenshtein as lev
 import os
+import yaml
 
 configs_path = os.path.join(os.path.dirname(__file__), "aux")
```

Comment:
I am not a big fan of global variables. Can't you just put them inside the functions that are calling them?